src/vehicle.cpp

```diff
@@ -354,6 +354,14 @@ void vehicle::init_state( int init_veh_fuel, int init_veh_status )
         }
     }
 
+    // Reduce fuel in new vehicles over time
+    int fuel_reducing = get_option< int >( "VEHICLE_FUEL_REDUCING" );
+    float fuel_mult_time = 1.0f;
+    if( fuel_reducing > 0 && veh_status != 0 ) {
+        int current_day = to_days< int >( calendar::turn - calendar::time_of_cataclysm );
+        fuel_mult_time = exp( - 0.69f * current_day / fuel_reducing );
```

Comment:
Please declare the -0.69f as a static_const someplace so it's easy to see and reference.